drivers/java/templates/Test.java

```diff
@@ -33,8 +33,8 @@
 import java.util.Collections;
 import java.nio.charset.StandardCharsets;
 
-import static gen.TestingCommon.*;
-import gen.TestingFramework;
+import static com.rethinkdb.gen.TestingCommon.*;
+import com.rethinkdb.gen.TestingFramework;
```

Comment:
This is the template where the generated tests get their imports from. If there's a reduced set of imports that can be used, it should be modified here